modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb

```diff
@@ -265,6 +244,51 @@ def write_output_files(rhost, username, sha1_salt, sha1_hash)
     end
   end
 
+  def service_data
+    {
+      address: rhost,
+      port: rport,
+      service_name: 'ipmi',
+      protocol: 'udp',
+      workspace_id: myworkspace_id
+    }
+  end
+
+  def report_hash(user, hash)
+    credential_data = {
+      module_fullname: self.fullname,
+      origin_type: :service,
+      private_data: hash,
+      private_type: :nonreplayable_hash,
+      jtr_format: 'rakp_hmac_sha1_hash',
```

Comment:
Again, I'm not sure i it's the format name which jhon the ripper uses. I copied it from the old `report_auth_info` call.